tests/cephadm/test_cephadm_pull.py

```diff
@@ -0,0 +1,70 @@
+import ast
+
+import requests
+import yaml
+
+from ceph.ceph_admin import CephAdmin
+from utility.log import Log
+
+log = Log(__name__)
+
+
+def getting_ceph_version(link):
+    """
+    Method return ceph version from magna002 latest ceph container yaml
+    Args:
+        link: magna002 latest ceph container yaml link
+    Returns:
+        ceph version
+    """
+
+    response = requests.get(link)
+    yaml_response = yaml.safe_load(response.text)
+    ceph_version_5 = []
+    for ceph_version in yaml_response:
+        if ceph_version >= 5:
+            ceph_version_5.append(yaml_response[ceph_version]["ceph-version"])
+    return ceph_version_5[-1]
+
+
+def run(ceph_cluster, **kw):
+    """
+    Args: Check the latest image version using cephadm pull option
```

Comment:
Remove this descripion. It's common method no need to add args and returns. Add descrioption for test case instead.

Reply:
Thanks @vamahaja
Removed